lapack/netlib/lapack.go

```diff
@@ -3274,6 +3274,52 @@ func (impl Implementation) Dsytrd(uplo blas.Uplo, n int, a []float64, lda int, d
 	lapacke.Dsytrd(byte(uplo), n, a, lda, d, e, tau, work, lwork)
 }
 
+// Dtbtrs solves a triangular system of the form
+//  A * X = B   if trans == blas.NoTrans
+//  Aᵀ * X = B  if trans == blas.Trans or blas.ConjTrans
+// where A is an n×n triangular band matrix with kd super- or subdiagonals, and
+// B is an n×nrhs matrix.
+//
+// Dtbtrs returns whether A is non-singular. If A is singular, no solutions X
```

Comment:
This reflects the gonum implementation, but "If A is singular, no solutions X are computed." should probably be either "If A is singular, no solutions for X are computed." or "If A is singular, no solution for X is computed." depending on your view of whether zero is singular in English.

Reply:
Changed to "no solution X is computed". The reasoning behind using X in plural is that solving a system with a matrix right-hand side with `nrhs` columns is like solving the system `nrhs` times with the _solutions_ stored in columns of X. I took it over from the Lapack documentation but I'm fine with "solution X". I'll change it also in the gonum implementation.